src/main.c

```diff
@@ -0,0 +1,308 @@
+/**
+ * @file main.c
+ * @author David Chung 
+ * @brief Main entry point for simple-calc.
+ * 
+ * This program performs simple arithmetic calculations given user input via CLI.
+ * 
+ * @version 3.1
+ * @date 2024-11-25
+ *
+ */
+
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdbool.h>
+#include <errno.h>
+#include <string.h>
+#include "arithmetic.h"
+#include "debug.h"
+#include "helpers.h"
+
+typedef enum
+{
+    ADD,
+    SUBTRACT,
+    MULTIPLY,
+    DIVIDE,
+    INVALID
+} Operation;
+    
+Operation checkOperator(const char *operator)
+{
+    if (strncmp(operator, "+") == 0)
```

Comment:
Check man page for strncmp, it should take the strings to compare and a length (3 arguments)